TestApp/ios/TestApp/AppDelegate.m

```diff
@@ -26,6 +27,7 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
+  [AppCenterReactNativeData register];  // Initialize AppCenter data
```

Comment:
For readability, let's move this line to line 42 so that all `register` calls are kept close to each other.